src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -134,3 +134,18 @@ implication        ,ARM64 ,Sha1      ,ArmBase
 implication        ,ARM64 ,Sha256    ,ArmBase
 implication        ,ARM64 ,Vector64  ,AdvSimd
 implication        ,ARM64 ,Vector128 ,AdvSimd
+
+
+;                    ,name and aliases           ,archs    ,lower baselines included by implication
+;
+instructionsetgroup  ,x86-x64                    ,X64 X86  ,sse2
+instructionsetgroup  ,x86-x64-v2                 ,X64 X86  ,sse4.2 popcnt
+instructionsetgroup  ,x86-x64-v3                 ,X64 X86  ,x86-x64-v2 avx2 bmi bmi2 lznct movbe fma
+instructionsetgroup  ,skylake                    ,X64 X86  ,x86-x64-v3 pclmul aes
+
+instructionsetgroup  ,armv8-a                    ,ARM64    ,neon
+instructionsetgroup  ,armv8-1-a                  ,ARM64    ,armv8-a lse crc rdma sha1 sha2 aes
```

Comment:
Nit: Does `armv8-1-a` map to any existing prior art? The arm typically uses `.` to separate major and minor version, like 
`armv8.1-a`